modules/gltf/src/glb/gltf-types.js

```diff
@@ -0,0 +1,31 @@
+// TODO - Remove: glTF constants should not be know to the GLB layer
```

Comment:
I think we should move this to luma.gl

Please remove if not referenced here.

Reply:
> I think we should move this to luma.gl. Please remove if not referenced here.

Problem is they are referenced here. I have a bullet in the GLTF Tracker to refactor this.